### DISCUSSION TITLE
Skip scheduled Run

### DISCUSSION THREAD

**Discussion by sou-joshi** [apache/airflow#17730] [Q&A]
I have Dag scheduled to run from 5-10th on every month. When it runs on 5th and is successful, I would like to skip all the following scheduled runs for the month and only trigger it next month. How can this be achieved? Please help

**Comment by BasPH** [comment]
Airflow does not support such dynamic conditions. My workaround would be to start with a task in your DAG which checks if it's the 5th of the month, and if not, checks the status of the run of the 5th and skips remaining tasks if the run was successful.

I haven't implemented something like this myself, but I think you can achieve it in different ways:
- read metastore directly, querying for DAGrun info
- using [ExternalTaskSensor](https://airflow.apache.org/docs/apache-airflow/stable/_api/airflow/sensors/external_task/index.html#airflow.sensors.external_task.ExternalTaskSensor)
